tools/CACertificates/certGen.sh

```diff
@@ -404,16 +423,30 @@ function generate_edge_server_certificate()
 }
 
 ###############################################################################
-# Generates a certificate for a Edge device, chained to the intermediate.
+# Generates a CA certificate for a Edge device, chained to the intermediate.
 ###############################################################################
-function generate_edge_device_certificate()
+function generate_edge_device_ca_certificate()
```

Comment:
Why did you remove parameter checking on this function? I ask because the previous function (`generate_edge_device_identity_certificate`) has parameter checking...

Reply:
The validations were not put in here and other the function. I have gone ahead and added these. Thanks.